flexmeasures/data/models/generic_assets.py

```diff
@@ -54,10 +54,17 @@ class GenericAsset(db.Model, AuthModelMixin):
     latitude = db.Column(db.Float, nullable=True)
     longitude = db.Column(db.Float, nullable=True)
     attributes = db.Column(MutableDict.as_mutable(db.JSON), nullable=False, default={})
+    parent_generic_asset_id = db.Column(
+        db.Integer, db.ForeignKey("generic_asset.id"), nullable=True
+    )
+    parent_generic_asset = db.relationship(
+        "GenericAsset", remote_side=[id], backref="child_generic_assets"
+    )
 
     generic_asset_type_id = db.Column(
         db.Integer, db.ForeignKey("generic_asset_type.id"), nullable=False
     )
+
```

Comment:
Let's adjust the grouping (i.e. using line breaks) according to the type of relationship, as follows:

```
# No relationship
db.Column(...)
db.Column(...)

# One-to-many (or many-to-one?) relationships
db.Column(..., db.ForeignKey, ...)
db.relationship(..., remote_side=... or foreign_keys=...)

# Many-to-many relationships
db.relationship(..., secondary=...)
```

Reply:
Good idea! 
Should I add the comments as well?